lib/ReactViews/Workbench/Controls/ColorScaleRangeSection.jsx

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+import React from 'react';
+import ObserveModelMixin from '../../ObserveModelMixin';
+
+import Styles from './colorscalerange-section.scss';
+
+const ColorScaleRangeSection = React.createClass({
+    mixins: [ObserveModelMixin],
+
+    propTypes: {
+        item: React.PropTypes.object.isRequired
+    },
+
+    minRange: -50,
+    maxRange: 50,
+
+    updateRange() {
+        this.props.item.parameters.colorscalerange = [this.minRange, this.maxRange].join();
+        this.props.item.refresh();
+    },
+
+    changeRangeMin(event) {
+        this.minRange = event.target.value;
+    },
+
+    changeRangeMax(event) {
+        this.maxRange = event.target.value;
+    },
+
+    render() {
+        const item = this.props.item;
+        this.minRange = item.parameters.colorscalerange.split(',')[0];
+        this.maxRange = item.parameters.colorscalerange.split(',')[1];
+        if (!item.isNcWMS) {
```

Comment:
This check needs to be moved earlier, and you should probably check for the presence of `colorscalerange` parameter, too, in case it's possible for a server we think is ncWMS to to not have it.